### ISSUE TITLE
Can I change data-default-file with jQuery ?

### ISSUE THREAD

**Issue by mecjos** [JeremyFagis/dropify#63]
I want to edit table element which includes image file. 
To do this, first I get existing file name with;

existingImage = $("#imgElement").attr("src");

then I try to change input element data-default-file attribute with;

$("#inputImageElement").attr("data-default-file", existingImage);

In that way, I can see data-default-file when I inspect element but I can't see on screen.

**Comment by roberto-slopez** [comment]
I use 

```js 
dropify.dropify({
           "defaultFile": "{{ asset('uploads/' ~ currentFileName ) }}",
            "messages": {
                    default: 'Arrastre un archivo o haga clic aquí',
                    replace: 'Arrastre un archivo o haga clic en reemplazar',
                    remove: 'Eliminar',
                    error: 'Lo sentimos, el archivo demasiado grande'
                }
            });
```

**Comment by JackZeled0n** [comment]
@roberto-slopez what do you want to say wit this?
`asset('uploads/' ~ currentFileName )`
Explain please

**Comment by roberto-slopez** [comment]
@JackZeled0n Sorry, I updated my comment

**Comment by JackZeled0n** [comment]
My team & me use angular, our solution were
`

    $(document).ready(function(){
        var nameImage =  'full path file';
        $('.dropify').dropify({
            defaultFile: nameImage ,
        });
    });
`

**Comment by hanhduykaka** [comment]
How can I binding image by Base64?

**Comment by JackZeled0n** [comment]
@hanhduykaka Are you using any specific function for making the Base 64?
I think something, you can get the image wherever you have it and you can decode for putting on
`defaultFile=your-image-decode.format(jpg,png)` 
the property by Dropify.

**Comment by hanhduykaka** [comment]
@JackZeled0n : You mean I must using [decode base64](https://stackoverflow.com/questions/2820249/base64-encoding-and-decoding-in-client-side-javascript) before using defaultFile and using when I want to binding code from the database?

**Comment by JackZeled0n** [comment]
@hanhduykaka Yes, you could use it for making the encode and decode the url of the image that you want put on the property.  I don't understand so good that you want to say with 

> I want to binding code from the database

**Comment by hanhduykaka** [comment]
@JackZeled0n : I have a form and using Dropify for upload Avatar. So when I get an API from backend and the data for Image is base64. I'm trying decode URL to blob but not working!

**Comment by mecjos** [comment]
@JackZeled0n I tried your solution. It didn't work. My dropify is in a edit modal form.

**Comment by JackZeled0n** [comment]
@mecjos can you show how you do?

**Comment by JackZeled0n** [comment]
@hanhduykaka did you find a solution? Sorry I didn't read the notification

**Comment by mecjos** [comment]
@JackZeled0n I do it as follow. 

```
$(document).ready(function() {
    $(document).on('click', '.editButton', function() {
       
        $.ajax({
            type: 'GET',
            url: 'projects/' + editProjectId + '/edit',
            dataType: 'json',
            error: function (data) {
                console.error('Error:', data);
            },
            success: function (response) {
                var nameImage = response.imagName
               $(document).ready(function(){
                  $('.dropify').dropify({
                       defaultFile: nameImage ,
                  });
               });

            }
        });
}
```

**Comment by JackZeled0n** [comment]
What brings the response? Make a console log and show us the mistake

`console.log(response.imagName);`

You have to be sure you get the good response from the request.

**Comment by akshaysalekar108** [comment]
@mecjos  Any Success buddy. I am Stuck at same issue , Please tell me how did you solve this problem :)

**Comment by hanhduykaka** [comment]
@akshaysalekar108 :
  defaultFile: blobFunction() ,

You can create a function: blobFunction() and return decode URL to blob from your base64

like [this](https://stackoverflow.com/questions/16245767/creating-a-blob-from-a-base64-string-in-javascript)

**Comment by developerabdan** [comment]
@hanhduykaka i'm trying this right now, but didn't solve my issue

**Comment by SkinnyCoders** [comment]
@hanhduykaka thanks dude, it's work fine

**Comment by hanhduykaka** [comment]
you're welcome
